skaffold.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: skaffold/v2alpha2
+kind: Config
+
+build:
+  tagPolicy:
+    sha256: {}
+  artifacts:
+    - image: shopozor/assets-fixtures
+      context: .
+      docker:
+        dockerfile: backend/assets-service/Dockerfile
+    - image: shopozor/graphql-engine
+      context: backend/database-service
+      docker:
+        dockerfile: Dockerfile
+        target: hasura-migrations
+    - image: shopozor/fixtures-service
+      context: .
+      docker:
+        dockerfile: backend/fixtures-generator/Dockerfile
+        target: app
+
+deploy:
+  helm:
+    releases:
+      - name: minio
+        chartPath: bitnami/minio
+        remote: true
+        namespace: dev
+        overrides:
+          fullnameOverride: minio
+          mode: standalone
+          clusterDomain: cluster.local
+          persistence:
+            enabled: false
+            size: 10Gi
+          disableWebUI: false
+          accessKey:
+            password: minio
+          secretKey:
+            password: minio123
+      # TODO: we need to decide if the assets are pushed by default to the assets service
+      #       or if we proceed as with the database fixtures!
+      - name: minio-initialization
+        chartPath: backend/assets-service/chart
+        namespace: dev
+        values:
+          image: shopozor/assets-fixtures
+        overrides:
+          fixtures:
+            enabled: true
+          minio:
+            fullname: minio
+            client:
+              alias: minio
+            server:
+              port: 9000
+      - name: postgres
+        chartPath: bitnami/postgresql-ha
```

Comment:
I propose to stick with this high-available postgres provided by bitnami in a first phase, even for production. We can always add some more monitoring and machinery later, with e.g. patroni or stolon.